xmake/rules/qt/xmake.lua

```diff
@@ -164,7 +164,13 @@ rule("qt.widgetapp_static")
             end
         elseif target:is_plat("wasm") then
             plugins.QWasmIntegrationPlugin = {linkdirs = "plugins/platforms", links = {"qwasm"}}
-            table.join2(frameworks, "QtEventDispatcherSupport", "QtFontDatabaseSupport", "QtEglSupport")
+            if qt_sdkver then
+                if qt_sdkver:lt("6.0") then
```

Comment:
if qt_sdkver and qt_sdkver:ge("6.0") then 
    opengl
else
    qtevent frameworks..
endif